start.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 DIR="$(cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
-cd "$DIR"
+cd "$DIR" || exit 1
```

Comment:
I'm all for fail-fast, but this seems pointless. The script isn't going to fail to cd into its own directory.